acceptance/tests/peering/peering_connect_test.go

```diff
@@ -90,14 +95,13 @@ func TestPeering_Connect(t *testing.T) {
 				staticServerPeerHelmValues["server.exposeGossipAndRPCPorts"] = "true"
 				staticServerPeerHelmValues["meshGateway.service.type"] = "NodePort"
 				staticServerPeerHelmValues["meshGateway.service.nodePort"] = "30100"
-				staticServerPeerHelmValues["server.exposeService.type"] = "NodePort"
```

Comment:
QQ: why are we removing the grpc port configs for each of these?

Reply:
We no longer need the exposeService at all for peering connections, so this removes any configuration of it.